File: app/src/main/java/org/stepic/droid/storage/structure/DbStructureStep.java
package org.stepic.droid.storage.structure;

@Deprecated
public final class DbStructureStep {
    public static final String STEPS = "steps";

    public static final class Column {
        public static final String STEP_ID = "step_id";
        public static final String LESSON_ID = "lesson_id";
        public static final String STATUS = "status";
        public static final String PROGRESS = "progress";
        public static final String SUBSCRIPTIONS = "subscription";
        public static final String VIEWED_BY = "viewed_by";
        public static final String PASSED_BY = "passed_by";
        public static final String CREATE_DATE = "create_date";
        public static final String UPDATE_DATE = "update_date";
        public static final String POSITION = "position";
        public static final String IS_CACHED = "is_cached";
        public static final String IS_LOADING = "is_loading";
        public static final String DISCUSSION_COUNT = "discussion_count";
        public static final String DISCUSSION_ID = "discussion_id";
        public static final String PEER_REVIEW = "has_peer_review";
        public static final String HAS_SUBMISSION_RESTRICTION = "has_submission_restriction";
        public static final String MAX_SUBMISSION_COUNT = "max_submission_count";
    }
}

File: app/src/main/java/org/stepik/android/cache/step/structure/DbStructureStep.kt
package org.stepik.android.cache.step.structure

object DbStructureStep {
    const val TABLE_NAME = "step"

    object Column {
        const val ID = "id"
        const val LESSON_ID = "lesson_id"
        const val POSITION = "position"

        const val STATUS = "status"
        const val PROGRESS = "progress"

        const val SUBSCRIPTION = "subscription"

        const val VIEWED_BY = "viewed_by"
        const val PASSED_BY = "passed_by"
        const val WORTH = "worth"

        const val CREATE_DATE = "create_date"
        const val UPDATE_DATE = "update_date"

        const val DISCUSSION_COUNT = "discussion_count"
        const val DISCUSSION_PROXY = "discussion_proxy"
        const val DISCUSSION_THREADS = "discussion_threads"
        const val PEER_REVIEW = "has_peer_review"
        const val HAS_SUBMISSION_RESTRICTION = "has_submission_restriction"
        const val MAX_SUBMISSION_COUNT = "max_submission_count"
        const val CORRECT_RATIO = "correct_ratio"
    }
}